src/DigitalSubscriberAppBanner.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from "react";
+import { DigitalSubscriberAppBanner } from "./DigitalSubscriberAppBanner";
+import { StorybookWrapper } from './utils/StorybookWrapper';
+
+export default {
+  component: DigitalSubscriberAppBanner,
+  title: "Components/DigitalSubscriberAppBanner",
+};
+
+export const defaultStory = (): ReactElement => {
+  return (
+    <StorybookWrapper>
+        <DigitalSubscriberAppBanner
+            onButtonClick={() => {
+                console.log("Button clicked!");
+            }}
+            message="Test message"
+            firstName="John"
+        />
+    </StorybookWrapper>
+  );
+};
+
+defaultStory.story = { name: "Sign In Component" };
```

Comment:
Looks like this name might be copied from a different component?

Reply:
Good spot - I renamed the component today and missed that one.